stopcovid/event_distributor/outbound_sms.py

```diff
@@ -24,31 +24,38 @@
 # We should template these for localization
 CORRECT_ANSWER_COPY = "Correct!"
 
+DELAY_BETWEEN_MESSAGES_IN_SECONDS = 3
```

Comment:
check with Kai on what this should actually be

Reply:
Will do. Figured we could configure it once we are testing flows end to end